File: PHP/Cap03/c03ex03.php
<?php
// c02ex03.php

printf("CALCULO DE SALARIO LIQUIDO\n\n");

printf("Entre quantidade de horas trabalhadas: ");
$HT = floatval(trim(fgets(STDIN)));

printf("Entre valor do salario por hora: ");
$VH = floatval(trim(fgets(STDIN)));

printf("Entre valor percentual de desconto: ");
$PD = floatval(trim(fgets(STDIN)));

$SB = $HT * $VH;
$TD = ($PD / 100) * $SB;
$SL = $SB - $TD;

printf("\n");
printf("Salario Bruto ....: %f\n", $SB);
printf("Salario Liquido ..: %f\n", $SL);

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>

File: SWIFT/Cap03/c03ex03.swift
// c03ex03.swift

import Foundation

var HT, VH, PD, SB, TD, SL: Double

print("Entre quantidade de horas trabalhadas ..: ", terminator: "")
if let inputHT = readLine(), let doubleHT = Double(inputHT) {
    HT = doubleHT
} else {
    fatalError("Entrada invalida para quantidade de horas trabalhadas")
}

print("Entre valor do salario-hora ............: ", terminator: "")
if let inputVH = readLine(), let doubleVH = Double(inputVH) {
    VH = doubleVH
} else {
    fatalError("Entrada invalida para valor do salário-hora")
}

print("Entre valor do percentual de desconto ..: ", terminator: "")
if let inputPD = readLine(), let doublePD = Double(inputPD) {
    PD = doublePD
} else {
    fatalError("Entrada invalida para valor do percentual de desconto")
}

SB = HT * VH
TD = (PD / 100) * SB
SL = SB - TD

print("\n")
print("Salario Bruto ....: \(SB)")
print("Salario Liquido ..: \(SL)")

print("\nTecle <Enter> para encerrar... ", terminator: "")
_ = readLine()
